src/components/executive-education-2u/ExecutiveEducation2UError.jsx

```diff
@@ -48,9 +76,21 @@ const ExecutiveEducation2UError = ({ failureReason, httpReferrer }) => {
                 global.location.href = httpReferrer;
               }}
             >
-              Return to your learning platform
+              Return to dashboard
             </Button>
           )}
+          {showHelpfulLink(failureReason) && (
+            <div>
+              <span className="executive-education-2u-error-link-description">
+                Helpful Link:
+              </span>
+              <span className="executive-education-2u-error-link-box">
+                <Button variant="link" size="sm">
+                  <ContactAdminMailto class="executive-education-2u-error-link" />
```

Comment:
`class` -> `className`, though it appears this class name is unused / not defined. Also note, the `ContactAdminMailto` component does not currently accept a `className` prop.